### PR TITLE
Poc to reuse MenuBlock

### PR DESCRIPTION
**Pull request by VincentLanglet**
## Subject

I am targeting this branch, because {reason}.

Closes #{put_issue_number_here}.

## Changelog

```markdown
### Fixed
- Revert BC break on BreadcrumbMenuBlockService
```

### CODE REVIEW

File: src/Block/Breadcrumb/BaseBreadcrumbMenuBlockService.php [hunk 1]
```diff
@@ -16,96 +16,61 @@
...
+                    'translation_domain' => 'SonataSeoBundle',
+                ]],
+                ['current_uri', TextType::class, [
+                    'required' => false,
+                    'label' => 'form.label_current_uri', // TODO
+                    'translation_domain' => 'SonataSeoBundle',
+                ]],
+                ['context', TextType::class, [
+                    'required' => false,
+                    'label' => 'form.label_context', // TODO
```
Comment (jordisala1991) [inline src/Block/Breadcrumb/BaseBreadcrumbMenuBlockService.php]: Those 3 translations are missing, do you plan to add them on this PR?

**Review by jordisala1991 — DISMISSED** [review]
Missing translations and missing release for block bundle first but 👍

File: src/Block/Breadcrumb/BaseBreadcrumbMenuBlockService.php (hunk 1 above)
Comment (VincentLanglet) [reply src/Block/Breadcrumb/BaseBreadcrumbMenuBlockService.php]: Translations are added

**Comment by VincentLanglet** [comment]
Should we release this as a patch or a minor @jordisala1991 ?

**Comment by jordisala1991** [comment]
Minor I would say

**Comment by VincentLanglet** [comment]
> Minor I would say

Ok ! Just waiting for your review when you have time :)

**Review by jordisala1991 — APPROVED** [review]

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**